admin/src/pages/List.jsx

```
@@ -0,0 +1,84 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { backendUrl, currency } from "../App";
+import { toast } from "react-toastify";
+
+const List = ({ token }) => {
+  const [list, setList] = useState([]);
+
+  const fetchList = async () => {
+    try {
+      const res = await axios.get(backendUrl + "/api/product/list");
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)